tests/tiup-cluster/script/scale_core.sh

```diff
@@ -62,13 +64,38 @@ function scale_core() {
     # validate https://github.com/pingcap/tiup/issues/786
     # ensure that this instance is removed from the startup scripts of other components that need to rely on PD
     ! tiup-cluster $client exec $name -N $ipprefix.101 --command "grep -q $ipprefix.103:2379 /home/tidb/deploy/tidb-4000/scripts/run_tidb.sh"
+
     echo "start scale out pd"
     topo=./topo/full_scale_in_pd.yaml
     sed "s/__IPPREFIX__/$ipprefix/g" $topo.tpl > $topo
     tiup-cluster $client --yes scale-out $name $topo
-
     # after scalue-out, ensure this instance come back
     tiup-cluster $client exec $name -N $ipprefix.101 --command "grep -q $ipprefix.103:2379 /home/tidb/deploy/tidb-4000/scripts/run_tidb.sh"
 
+    echo "start scale in node-2 with [tidb, tikv]"
+    tiup-cluster $client --yes scale-in $name -N $ipprefix.102:4000
+    wait_instance_num_reach $name $total_sub_one $native_ssh
+    tiup-cluster $client --yes scale-in $name -N $ipprefix.102:20160
+    echo "after sclae in(without prune) tikv, the monitors should not be destroyed"
+    wait_instance_num_reach $name $total_sub_one $native_ssh
```

Comment:
> after sclae in(without prune) tikv, the monitors should not be destroyed

`waiit_instance_num_reach` will call prune automaticlly, this is not what you expected

Reply:
but why it was passed in my local dev

Reply:
I'll dig into later tonight, and thanks for your help 😀

Reply:
It passed in your local dev, I think it may related to the opportunity:
- Only when the tikv's state is tombstone, the prune command cleanup it
- After scale in a tikv node, it will take some time to transfer to tombstone state (seconds to minutes)
So I guess your local host, when the `wait_instance_num_reach` execute, the tikv node is still in `pending offline` state, so it passed, but in CI, the tikv node is `tombstone`, so it was cleanup.

Reply:
1. Seems in Github CI, it's hard to predicate when TiKV will finish the transferring process and go to tombstone state, after that `prune` can evict the TiKV. 
2.  have reserved only one TIDB node on the 102 node for testing the issue #842